### DISCUSSION TITLE
How to check if my pi4 with raspbian 64 bit is farming?

### DISCUSSION THREAD

**Discussion by andre-bender** [Chia-Network/chia-blockchain#1887] [Q&A]
Hey everyone, I just can't find any commands to check whether my pi is farming. I have raspbian 64 bit installed and use the CLI version, since the GUI version doesn't open after installation.

Unfortunately I can't find any way to check the farming status. In the Win 10 GUI I've noticed, that the status "farming" doesn't mean it's actually farming. Because there are several bugs right now where "Last attempted proof" isn't refreshing and therefore the farming doesn't happen. To avoid downtime from my farmer I would like to check the plot filter, but I don't see a command in the CLI version.

Do you have any tips?

**Comment by norbul** [comment]
Try:
`chia farm summary`

You should get 
`Farming status: Farming`

**Reply by andre-bender** [reply]
Thank you, but as mentioned above the farming status isn't always telling the truth. It would be nice to check the plots passing the filter via cli commands.

**Reply by norbul** [reply]
... you can check your plots:

`chia plots check`

**Reply by andre-bender** [reply]
With that command I know that my plots are valid. It still doesn't tell me that they are passing the plot filter to the actual time. ;)

**Reply by hoffmang9** [reply]
Set your logging to INFO and then you can do this: 
```
tail -F ~/.chia/mainnet/log/debug.log | grep 'were.eligible.for.farming\|ERROR\|WARNING\|Finished.signage.point\|wallet.peak.to\|peak.to.height\|🍀'
```

You will see the same basic data as the GUI.

**Reply by norbul** [reply]
Change in config yaml 
log_level: INFO

and then in logs look up for:
 `1 plots were eligible for farming XXXXX... Found 0 proofs. Time: 2.03555 s. Total XXX plots`

**Reply by andre-bender** [reply]
> Set your logging to INFO and then you can do this:
> 
> ```
> tail -F ~/.chia/mainnet/log/debug.log | grep 'were.eligible.for.farming\|ERROR\|WARNING\|Finished.signage.point\|wallet.peak.to\|peak.to.height\|🍀'
> ```
> 
> You will see the same basic data as the GUI.

Thanks gene, as always ;)

**Reply by cciollaro** [reply]
try:
```
tail -f ~/.chia/mainnet/log/debug.log | grep harvester
```

if you run this, you'll see log lines like the following start to pop up
```
2 plots were eligible for farming xxxxxx... Found 0 proofs
```

NOTE: you'll need to set log level to INFO in ~/.chia/mainnet/config/config.yaml

**Reply by cciollaro** [reply]
oops somehow I missed the posts above

**Comment by andre-bender** [comment]
Hey guys, I need again your help. I setup my Pi now but I think it isn't farming properly. I get the following Error messages with your steps from above. I get the messages "X plots were eligible for farming...." but I also get the following "SSL error in data received" messages.

2021-04-27T09:35:13.513 full_node chia.full_node.full_node: INFO     ⏲️  Finished signage point 12/64: ....
2021-04-27T09:35:15.001 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming XXXX... Found 0 proofs. Time: 1.41866 s. Total 833 plots
2021-04-27T09:35:15.955 full_node asyncio                 : ERROR    SSL error in data received
2021-04-27T09:35:16.200 full_node asyncio                 : ERROR    SSL error in data received
2021-04-27T09:35:19.984 full_node asyncio                 : ERROR    SSL error in data received
2021-04-27T09:35:20.164 full_node asyncio                 : ERROR    SSL error in data received
2021-04-27T09:35:20.301 full_node asyncio                 : ERROR    SSL error in data received

I opened Port 8444 and disabled upnp on my Pi. But if I check with `chia farm summary` "Last farmed height" is stuck at XXXX. Is that normal and upgrades first when I win at this block height?

**Reply by dorkmo** [reply]
Yes last height doesnt change until you win again. (last height is usually kept private). Normally people check the plot filter to confirm everything is running.

**Reply by dorkmo** [reply]
not sure about the ssl error, i think someone was having a similar issue, you might search.

**Comment by rcamarda390** [comment]
did you check: https://github.com/Chia-Network/chia-blockchain/wiki/How-to-Check-If-Everything-is-Working-(or-Not)
 tail ~/.chia/mainnet/log/debug.log | grep eligible
